app/src/debug/res/mipmap-anydpi-v26/ic_launcher.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="utf-8"?>
+<adaptive-icon xmlns:android="http://schemas.android.com/apk/res/android">
+    <background android:drawable="@drawable/ic_launcher_background"/>
+    <foreground android:drawable="@drawable/ic_launcher_foreground"/>
```

Comment:
May you please add the monochrome icon too which was introduced with Android 13?
<monochrome android:drawable="@drawable/ic_launcher_monochrome"/>

Reply:
Sure, I will commit that in a next pull.